app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
+    total = 0
+
```

Comment:
1) No need to use blank line after a 'total' variable but not a mistake. Blank spot used sparingly inside functions to show clear steps, 'total' relates to loop block though.